code/modules/research/stock_parts.dm

```diff
@@ -313,6 +313,12 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	desc = "A large piece of equipment used to open a window into the subspace dimension."
 	materials = list(/datum/material/iron=50)
 
+/obj/item/stock_parts/card_reader
+	name = "card reader"
+	icon_state = "card_reader"
+	desc = "A small magnetic card reader, used for devices that take and transmit holocredits."
+	custom_materials = list(/datum/material/iron=50, /datum/material/glass=10)
```

Comment:
Are any more uses planned for this item or is it going to go the way of the console screen, utterly useless beyond a few recipes.

Reply:
No